bin/www

```diff
@@ -37,4 +40,17 @@ server.on('listening', () => {
   console.log(`Listening on ${bind}`)
 })
 
+//TODO: Verify this area.
 module.exports = app
```

Comment:
Hi @kevinpr007,
Did you manage to test all our scripts after removing this line of code?